csharp/ReadFile.cs

```diff
@@ -0,0 +1,21 @@
+
+// Read a file from file path (given as a command line arg),
+// print line by line with line numbers
+using System;
+using System.IO;
+using System.Linq;
+
+class ReadFile
+{
+    public static void Main (string[] args)
+    {
+        string filePath = args[0];
+        int lineNumber = 1;
+        var lines = File.ReadAllLines(filePath);
+        lines.Select(line => $"{lineNumber++} {line.ToUpper()}")
```

Comment:
```suggestion
        lines.Select((line, index) => $"{index + 1} {line.ToUpper()}")
```
if we do this, we can get rid of the `lineNumber` variable